src/lava/magma/runtime/runtime_services/runtime_service.py

```diff
@@ -345,9 +345,11 @@ def run(self):
                         if self.runtime_to_service.probe():
                             cmd = self.runtime_to_service.peek()
```

Comment:
I'm assuming `peek()` leaves the message in the queue and `recv()`. So it would be a problem if the message is not removed from the queue. 

Wouldn't it work to just call `recv()` here instead of peek?

Reply:
If we call recv, it will remove any other command as well. There can be command such as GET_DATA, SET_DATA.